src/main/java/org/jspecify/nullness/ImpliesAnnotations.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2022 The JSpecify Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jspecify.nullness;
+
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Container annotation for the repeatable {@link Implies @Implies} annotation.
+ *
+ * <p><b>WARNING:</b> This annotation is under development, and <i>any</i> aspect of its naming,
+ * location, or design may change before 1.0. <b>Do not release libraries using this annotation at
+ * this time.</b>
+ */
+@Documented
+@Retention(RUNTIME)
+@Target(ANNOTATION_TYPE)
+public @interface ImpliesAnnotations {
```

Comment:
I like your case in https://github.com/jspecify/jspecify/issues/215#issuecomment-1205645618 that we should just avoid `@Repeatable`. But if we have it for now, maybe we'd want to nest this annotation under `@Implies` to make it less prominent and more clearly subservient to `@Implies`, similar how [`@EnsuresNonNullIf`](https://checkerframework.org/api/org/checkerframework/checker/nullness/qual/EnsuresNonNullIf.html) has a nested `@List` annotation? Maybe we should even copy the pattern of calling this "`List`" (though I guess that's an unfortunate simple name, so maybe not... but something simple?).

Reply:
I didn't know that was allowed!